src/mobility/components/CarSharingStationBottomSheet.tsx

```diff
@@ -33,8 +32,10 @@ type Props = {
 };
 
 export const CarSharingStationSheet = ({stationId, distance, close}: Props) => {
```

Comment:
Not a part of your changes, but to keep the name in sync with the file name, this component should be called `CarSharingStationBottomSheet`

Reply:
Updated